pkgs/by-name/br/bruno/scroll-width-fix.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/packages/bruno-app/src/globalStyles.js b/packages/bruno-app/src/globalStyles.js
+index 78c81b8..af4ae6b 100644
+--- a/packages/bruno-app/src/globalStyles.js
++++ b/packages/bruno-app/src/globalStyles-darwin.js
+@@ -163,32 +163,6 @@ const GlobalStyle = createGlobalStyle`
+   }
+
+
+-  // scrollbar styling
+-  // the below media query target non-macos devices
+-  // (macos scrollbar styling is the ideal style reference)
+-  @media not all and (pointer: coarse) {
+-    * {
+-      scrollbar-width: thin;
+-      scrollbar-color: ${(props) => props.theme.scrollbar.color};
+-    }
+-
+-    *::-webkit-scrollbar {
+-      width: 5px;
+-    }
+-
+-    *::-webkit-scrollbar-track {
+-      background: transparent;
+-      border-radius: 5px;
+-    }
+-
+-    *::-webkit-scrollbar-thumb {
+-      background-color: ${(props) => props.theme.scrollbar.color};
+-      border-radius: 14px;
+-      border: 3px solid ${(props) => props.theme.scrollbar.color};
+-    }
+-  }
+-
+-
+   // codemirror
+   .CodeMirror {
+     .cm-variable-valid {
+
```

Comment:
The build issue is caused by trimmed spaces in the patch file, adding them back fixes it.. Also why is `globalStyles.js` renamed to `globalStyles-darwin.js` does bruno even use that file if it's named differently?

```suggestion
diff --git a/packages/bruno-app/src/globalStyles.js b/packages/bruno-app/src/globalStyles.js
index c2b16781..936449ed 100644
--- a/packages/bruno-app/src/globalStyles.js
+++ b/packages/bruno-app/src/globalStyles.js
@@ -163,32 +163,6 @@ const GlobalStyle = createGlobalStyle`
   }
 
 
-  // scrollbar styling
-  // the below media query target non-macos devices
-  // (macos scrollbar styling is the ideal style reference)
-  @media not all and (pointer: coarse) {
-    * {
-      scrollbar-width: thin;
-      scrollbar-color: ${(props) => props.theme.scrollbar.color};
-    }
-    
-    *::-webkit-scrollbar {
-      width: 5px;
-    }
-    
-    *::-webkit-scrollbar-track {
-      background: transparent;
-      border-radius: 5px;
-    }
-    
-    *::-webkit-scrollbar-thumb {
-      background-color: ${(props) => props.theme.scrollbar.color};
-      border-radius: 14px;
-      border: 3px solid ${(props) => props.theme.scrollbar.color};
-    }
-  }
-
-
   // codemirror
   .CodeMirror {
     .cm-variable-valid {
```